runners/genlayer-py-std/src-emb/genlayer_embeddings/model_wrappers.py

```
@@ -86,13 +61,46 @@ def _unfold(x: np.ndarray):
 _cache_SentenceTransformer: dict[str, typing.Callable[[str], np.ndarray]] = {}
 
 
+def SentenceTransformerFromPath(path: str) -> typing.Callable[[str], np.ndarray]:
+	if res := _cache_SentenceTransformer.get(path):
+		return res
+	from word_piece_tokenizer import WordPieceTokenizer
+
+	tokenizer = WordPieceTokenizer()
+	data = Path(path).read_text()
+	globs = {}
+	exec(data, globs)
```

Ruff (0.12.2)

Use of `exec` detected

(S102)